src/lsp-server.ts

```diff
@@ -49,15 +50,17 @@ export class LspServer {
     }
 
     didChangeWorkspaceFolders(params: DidChangeWorkspaceFoldersParams) {
-        this.rootPath = fileURLToPath(params.event.added[0].uri);
-        this.runCtags(this.rootPath);
+        const rootPath = fileURLToPath(params.event.added[0].uri);
```

Comment:
I have question that if we delete a repository will code server send `didChangeWorkspaceFolder` request with the removed folders?

We just extract the first element of `params.event.added` here, is there a chance omit remain folders. In other way, is it a strong gurantee that `parasm.event.added` just has one element?

Reply:
Yeh, I missed that, I will add the removed workspace logic.

Reply:
Done.